addons/hr_maintenance/models/equipment.py

```diff
@@ -6,14 +6,15 @@
 class MaintenanceEquipment(models.Model):
     _inherit = 'maintenance.equipment'
 
-    employee_id = fields.Many2one('hr.employee', string='Assigned Employee', tracking=True)
-    department_id = fields.Many2one('hr.department', string='Assigned Department', tracking=True)
+    employee_id = fields.Many2one('hr.employee', compute='_compute_equipment_assign', store=True, readonly=False, copy=True, string='Assigned Employee', tracking=True)
+    department_id = fields.Many2one('hr.department', compute='_compute_equipment_assign', store=True, readonly=False, copy=True, string='Assigned Department', tracking=True)
```

Comment:
Please split the fields definition in 2 or more lines when they go over ~100 characters/line, for code readability.